webui/src/pages/repositories/wizard/spark_quickstart_wizard.jsx

```diff
@@ -45,48 +45,59 @@ const RepositoryCreationStep = ({repoCreationError, createRepo, onCancel, onComp
         content = <Alert variant="info" className={"mt-3"}>Repository <span className={"font-weight-bold"}>{repoName}</span> created successfully</Alert>;
     }
     else {
-        content = <RepositoryCreateForm config={response} error={showError} onSubmit={onSubmit} onCancel={onCancel}/>;
+        content = <RepositoryCreateForm config={response} error={showError} onSubmit={onSubmit} onCancel={onCancel} sm={8}/>;
     }
     return (
         <>
-            <h2 className={"wizard-step-header"}>Create a repository</h2>
+            <h3 className={"wizard-step-header"}>Create a repository</h3>
             {content}
         </>
     );
 }
 
 export const SparkQuickstart = ({onExit, createRepo, repoCreationError}) => {
-    const [nextEnabled, setNextEnabled] = useState(false)
     const [state, setState] = useState({
-        'branch': '',
-        'namespace': '',
-        'repoId': '',
+        branch: '',
+        namespace: '',
+        repoId: '',
     });
-    const completedStep = (val = {}) => {
-        setState({...state, ...val});
-        setNextEnabled(true);
+    const [completed, setCompleted] = useState(new Set());
+    const completedStep = (vals = {}, stepNum) => {
+        setState({...state, ...vals});
+        let newCompleted = new Set(completed.values());
+        newCompleted.add(stepNum);
+        setCompleted(newCompleted)
```

Comment:
```suggestion
        setCompleted(current => new Set(current).add(stepNum))
```

(and in some more places)